Logical Frameworks Chapter (#3430)
the-turing-way/the-turing-way: book/website/project-design/logical-frameworks-overview/theory-of-change.md

> 

Reviewer: We need a top level heading here: e.g. `# Something`

(pd-theory-of-change)=

> ## Theory of Change (ToC)
> 
> A theory of change helps you connect the dots between your actions and the positive change you want to see in the world. It's a powerful tool for planning and evaluating the effectiveness of your efforts, whether you're working on social causes, personal goals, or any other project. It encourages you to think critically about the steps needed to make a real impact.

Reviewer: ```suggestion
## Theory of Change

A theory of change (ToC) helps you connect the dots between your actions and the positive change you want to see in the world. It's a powerful tool for planning and evaluating the effectiveness of your efforts, whether you're working on social causes, personal goals, or any other project. It encourages you to think critically about the steps needed to make a real impact.
```

- **Definition**: Originating from theory-driven evaluation in the 1990s, ToC is an extensive methodology that illustrates complex pathways leading to a desired change or impact.
- **Nature**: Explanatory, presenting the comprehensive picture with all potential pathways to change.
- **Structure**: Adaptable, can be a flow chart or an intricate diagram with narrative text. Includes cyclical processes, feedback loops, and multiple pathways.
- **Focus**: Addresses broader aspects of change, including uncontrollable factors and the wider context of culture and power relations.
- **Process**: Involves deep critical reflection, logical thinking, and extensive data. It starts by identifying the goal and works backward, asking "if we do X, then Y will happen because…".
- **Usage**: Ideal for complex projects across multiple sectors, requiring rigorous planning and a thorough understanding of the intervention's impact.

> <img width="603" alt="Screenshot 2023-11-17 at 09 13 35" src="https://github.com/the-turing-way/the-turing-way/assets/53487593/380d3c0d-4c1a-4965-bba0-c4a16487437e">

Reviewer: 1. It may be better to use Markdown formatting for images here, e.g., `![alt text here](URL or path to file if it is in the the repo here)`
2. Please update the alt text

> ## Logical Framework (LogFrame)
> 
> A Logical Framework is like zooming in on the specific pathway that your program deals with and creating a neat, orderly structure for it. This makes it easier for you and the donor to monitor program implementation.

Reviewer: ```suggestion
## Logical Framework

A Logical Framework (Log Frame) is like zooming in on the specific pathway that your program deals with and creating a neat, orderly structure for it. This makes it easier for you and the donor to monitor program implementation.
```

This portmanteau also needs to be introduced in the logical-frameworks-overview.md file where it is used as well. See earlier confused comment :)

- **Definition**: Introduced in the late 1960s for USAID projects, LogFrame is a tool for project design, monitoring, and evaluation.
- **Nature**: Descriptive, focusing on a specific pathway from activities to outcomes.
- **Structure**: Standardised and linear, often presented as a table or flow chart without cyclical processes or feedback loops.
- **Focus**: Concentrates on the project itself, detailing a clear pathway from activities to outcomes.
- **Process**: Developed after project initiation, it follows a forward approach from inputs to goals, asking "if we plan to do A, will B happen?".
- **Usage**: Suitable for small to medium-sized projects, providing a clear and concise overview of project components.

> <img width="627" alt="Screenshot 2023-11-17 at 09 13 04" src="https://github.com/the-turing-way/the-turing-way/assets/53487593/96ad2ba6-c937-415a-bfd2-1bcce64c777d">

Reviewer: Again let's use Markdown formatting for images and improve the alt text please

> ### Similarities
> - Both are used for planning, monitoring, and evaluation.
> - They improve project design and outcomes.
> - Identify project components and demonstrate their relationships.
> - Represented visually for easier understanding.
> - Outcome-focused, based on logical assumptions.
> - Require regular updates to reflect changes.
> 
> ### Differences

Reviewer: ```suggestion
## Similarities
- Both are used for planning, monitoring, and evaluation.
- They improve project design and outcomes.
- Identify project components and demonstrate their relationships.
- Represented visually for easier understanding.
- Outcome-focused, based on logical assumptions.
- Require regular updates to reflect changes.

## Differences
```

It is worth not nesting the headings under the Logical Framework section here, since these sections are describing the Similarities and Differences between ToC and LogFrame, and hence don't explicitly belong to either section
